corehq/apps/userreports/transforms/custom/date.py

```diff
@@ -1,5 +1,13 @@
 import calendar
+from datetime import datetime
+from dimagi.utils.dates import force_to_datetime
 
 
 def get_month_display(month_index):
     return calendar.month_name[int(month_index)]
+
+
+def days_elapsed_from_date(date):
+    date = force_to_datetime(date)
+    now = datetime.utcnow()
+    return (now - date).days
```

Comment:
Oh I see - sorry we have two types of transforms and I was confused. This makes sense to me.